sonar-project.properties

```
@@ -0,0 +1,7 @@
+sonar.projectName=digital-marketplace
+sonar.sources=src/front-end,src/back-end
+sonar.sourceEncoding=UTF-8
+
+sonar.organization=bcgov-sonarcloud
+sonar.projectKey=digital_marketplace
+sonar.host.url=https://sonarcloud.io/
```

```suggestion
sonar.host.url=https://sonarcloud.io/

```
That icon in the diff is telling you [this file is missing its terminating newline](https://thoughtbot.com/blog/no-newline-at-end-of-file). @junminahn [used pre-commit in the sso-terraform repo](https://github.com/bcgov/sso-terraform/blob/ad9c56c2fc10a99b81577ec1406b2e9ed38c8410/.github/workflows/pre-commit.yml#L28) to avoid this error using the [end-of-file-fixer](https://github.com/bcgov/sso-terraform/blob/ad9c56c2fc10a99b81577ec1406b2e9ed38c8410/.pre-commit-config.yaml#L12) guardrail from [the official supported list](https://pre-commit.com/hooks.html).